lib/kubeclient/common.rb

```diff
@@ -127,8 +127,11 @@ def self.define_entity_methods(entity_types)
     end
 
     def self.pluralize_entity(entity_name)
-      return entity_name + 's' if entity_name.end_with? 'quota'
-      entity_name.pluralize
+      entity_name.end_with?('s') ? entity_name + 'es' : entity_name + 's'
```

Comment:
Prefer interpolation over `+`.  The latter creates extra strings which aren't needed.